src/main/java/com/minecolonies/coremod/entity/ai/citizen/builder/EntityAIStructureBuilder.java

```diff
@@ -77,7 +80,7 @@
     /**
      * After how many actions should the builder dump his inventory.
      */
-    private static final int ACTIONS_UNTIL_DUMP = 1024;
+    private static final int ACTIONS_UNTIL_DUMP = 4096;
```

Comment:
Why are we increasing this?

Reply:
if the Builder collected all materials for build we have  about 20 stacks in the inventory. which means if he is going to place all these 1280 blocks he will run back before placing all and dump the inventory. he will dump it anyway between the buildstages. also when he picks more material. I increased it to avoid this behaviour of unnescessary walks.